README.md

```diff
@@ -113,13 +113,13 @@ Currently the most reliable way of installing Gatekeeper is to build and install
 
 #### Deploying via Helm ####
 
-A basic Helm v2 template exists in `charts/gatekeeper`. If you have Helm installed and Tiller initialized on your cluster you can deploy via
+A basic Helm chart exists in `charts/gatekeeper`. If you have Helm installed, you can deploy via
 ```sh
 helm repo add gatekeeper https://open-policy-agent.github.io/gatekeeper/charts
-helm install gatekeeper/gatekeeper --devel
+helm install gatekeeper/gatekeeper --generate-name
```

Comment:
Looks like these instructions are defaulting to helm v3. Can you pls add that to the top?